os/binfmt/binfmt_exit.c

```diff
@@ -137,13 +137,14 @@ int binfmt_exit(FAR struct binary_s *bin)
 		}
 		address = (FAR void *)sq_next((FAR sq_entry_t *)address);
 	}
-	mm_remove_app_heap_list(bin->uheap);
+	mm_disable_app_heap_list(bin->uheap);
```

Comment:
please add mm_disable/remove_app_heap_list in exec_module when it is failed also.

Reply:
Done.